client/src/main/scala/org/scastie/client/ScastieBackend.scala

```diff
@@ -26,7 +30,12 @@ case class ScastieBackend(scastieId: UUID, serverUrl: Option[String], scope: Bac
   }
 
   val codeChange: String ~=> Callback =
-    Reusable.fn(code => scope.modState(_.setCode(code)))
+    Reusable.fn(code => {
+      scope.modState(state => {
+        val newState = state.setCode(code)
+        newState
+      })
+    })
```

Comment:
I think you can revert this change:

```suggestion
Reusable.fn(code => scope.modState(_.setCode(code)))
```